readme/git-pullrequest-workflow.md

```diff
@@ -10,26 +10,40 @@ Pick a name for your branch that tells us what it's for. Use this format:
 - `<issue number>`: This is the number of the task or issue in Github Projects you're working on.
 - `short-description`: This is a quick summary of what you're doing. Use hyphens instead of spaces.
 
+❗Please note - the `short-description` shouldn't contain numbers.
```

Comment:
Why can't we keep the numbers?

Reply:
I decided to remove the numbers from the description, since I did a simple parsing of the branch number - it looks for <number> in the branch name string.  I couldn't think of an option where we need numbers in the description